index.html

```diff
@@ -1,16 +1,120 @@
+    <main class="homepage-main">
+        <section class="homepage-hero">
+            <header>
+                <svg viewBox="0 0 100 80" width="40" height="40">
+                    <rect width="100" height="20"></rect>
+                    <rect y="30" width="100" height="20"></rect>
+                    <rect y="60" width="100" height="20"></rect>
+                </svg>
+                <img class="image" src="http://www.clker.com/cliparts/9/5/G/c/a/4/pink-frosted-cupcake-md.png" width="200px" alt="massive cake">
+                <h4>Cute Cakes</h4>
+                
+                <h3 class="h3header">The best cakes in town delivered to your door!</h3>
+                   <nav class="site-nav">
+                        <ul>
+                          <li><a href="#">Home</a></li>
+                          <li><a href="#">Cakes</a></li>
+                          <li><a href="#">Order</a></li>
+                          <li><a href="#">About</a></li>
+                          <li><a href="#">Contact</a></li>
+                        </ul>
+                    </nav>
+             </header>
+        </section>
+        
+        
+    <section class="homepage-feature"> 
+            <div class="welcome">
+                <h1 id="welcome">Welcome</h1>
+                <p>Hey! Glad to see you! Today is your day to taste the most delicious cakes and capcakes this side of heaven, look around, order and ENJOY!</p>
+                <button class="button">Order Now</button>
+            </div>
+            
+            <div class="featureimg">
+                <img src="https://media.istockphoto.com/photos/birthday-cake-decorated-with-colorful-sprinkles-and-ten-candles-picture-id1136810581?k=20&m=1136810581&s=612x612&w=0&h=Jj80oUK2CtY9nwYYRfJXB31ydjhOo5igRolQn-lV8M0=" alt="birthday celebtayion cake with sprinckles">
+            </div>
+    </section> 
+        
+        
+    <section class="homepage-products">
+           
+           <img src="img/istockphoto-1288470137-170667a.jpg" alt="pink cake image with sprincles">  
+           <img src="img/cakeshop3.png" alt="delicious wjite cupcakes">  
+           <img src="img/katie-rosario-QNyRp21hb5I-unsplash.jpg" width="509px" height="700px" alt="cake">    
+           <img src="img/heather-ford-Fq54FqucgCE-unsplash.jpg" width="509px" height="700px"alt="cake picture">
+           <img src="img/istockphoto-1346956805-170667a.jpg" width="509px" height="339px"alt="maccarons">
+           <img src="https://cdn.sallysbakingaddiction.com/wp-content/uploads/2019/07/vertical-layer-cake-265x265.jpg" width="509px" height="700px">
+    </section>
+
+
+
+        <aside class="homepage-aside">
+            <h5>We have a 10% promo code for new customers! Would you like to get it now?
+                <button class="button">Get Offer Now</button>
+            </h5>
+        </aside>
+    </main>
+      
+    
+    <footer class="footer">
+        <form>
+            <h2>Get In Touch</h3>
```

Comment:
I can see you have got a 'h2' tag that you closed with a 'h3'  closing tag.

Reply:
This was a mistake on my part, thanks for spotting that.